fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/BlockRegistry.java

```diff
@@ -28,6 +28,7 @@ private BlockRegistry() {
 	}
 
 	public static <T extends Block> T register(Identifier id, T block) {
+		block.setTranslationKey(id.toString());
```

Comment:
is this correct? isn't this some weird tile.modid.id.name

Reply:
that gets done automatically in `getTranslationKey`
`setTranslationKey` is only the id
in minecraft names like `notGate` are used